docs/pages/repo/docs/core-concepts/scopes.mdx

```diff
@@ -9,7 +9,7 @@ import HeartIcon from "@heroicons/react/solid/HeartIcon";
 
 <Callout type="error">
   `--scope` is deprecated in `1.2.x`. Please use
-  [`--filter`](/docs/core-concepts/filtering) instead.
+  [`--filter`](/repo/docs/core-concepts/filtering) instead.
```

Comment:
I think this is the page we want to link to?
```suggestion
  [`--filter`](/repo/docs/core-concepts/monorepos/filtering) instead.
```

Reply:
thanks!